### DISCUSSION TITLE
Discover Cardano - Take Two

### DISCUSSION THREAD

**Discussion by Tarrant64** [cardano-foundation/cardano-org#53] [Ideas]
**Problem:** The current Discover Cardano page doesn't really lend itself to newcomers on actually discovering Cardano in the way that we probably should.  

_First off, this is just an opinion, but hear me out._ Clicking on a link that says "Discover" Cardano shouldn't take you to a header that talks about principles, nexus, constellations, technology, research - you've already lost the user and they haven't started to scroll yet.  Discover Cardano should land on a page where we provide direct showcases to use cases on Cardano.  From books to mobile networks, real world assets being purchased to owning diamonds.  

Discover Cardano should land on use cases - projects alive and well building today and their use cases and what problems they're solving. Today, you can go to Cardano.org and only learn about it at a high-level without actually learning about what you can do in the ecosystem.  This doesn't keep attention where it ought to be.

**Initial thoughts on changes needed:**
At the header of the page, under Learn - "Discover Cardano" should just say "About". That's more or less what it is.  Discover Cardano should be a new link at the header of the page that takes you to a new page that showcases the projects on Cardano - but more specifically in a format that markets their use cases.

_"Cardano is..."_
- owning your own books and digital media
- new and open mobile network, internet for all.
- insert rolling list of use cases...

It's not a page to talk about blockchain, principles, technology - **it's a page to relate to the people** (not users - intentionally worded). TLDR - Should be a page for those that don't care what's under the hood, think normalish people - they just want to see it work and be able use it.

**Comment by Tarrant64** [comment]
Couple examples, because I think there are some good ideas out there:
Solana has a slogan that are running with - "Only possible on..."
![image](https://github.com/cardano-foundation/cardano-org/assets/126993855/a5b3ece8-798a-4042-93ac-6ad0fbc374a3)

Ethereum just simply links to some direct use cases in varios areas:
![image](https://github.com/cardano-foundation/cardano-org/assets/126993855/7fa701e4-2048-4cbc-a84e-e92193f9d9a3)

**Comment by katomm** [comment]
Thank you for starting this discussion. In order to conduct this discussion properly, one should also consider the content of this page https://cardano.org/enterprise. I personally believe that the use cases should be listed there and that the name should be changed from "Enterprise" to either "Enterprise use cases" or "Use cases".

**Reply by Tarrant64** [reply]
Thank you for the feedback.  To add onto your track there, I think the current information in /enterprises is still somewhat valuable to have - somewhere. And I agree it shouldn't be "Enterprises" regardless of what is there. It addresses "sectors" maybe more than anything. Still good information there I think to have, but not something to mix with what I'm proposing we see. 

So I'd almost suggest leaving it as it but changing the name, and having a new link altogether for the active projects/products that we want market - if that makes sense.

**Comment by RayTracedObserver** [comment]
Another possible solution would be to completely get rid of the discover cardano page and focus on making a very good home page on which visitors actually discover the main advantages and features of cardano without having to go to another page. Other pages should only be for more specific content that may not be of interest to a more general audience. The homepage should be how one discovers cardano.

What do you think about this option ?

**Reply by Tarrant64** [reply]
I kind of like that option - some thoughts on that too.  I think there's still something to discover about Cardano in general going to the home page.  And it, to me, is personally fine as it is right now.  Without creating too much work - perhaps we do scrap the Discover Cardano link - and merge that content elsewhere and rename it something that makes more sense.  The existing Discover Cardano section on the homepage should probably be replaced with content that has a slider show pictures of projects, conference events, etc. - show the very much alive and vibrant user base of Cardano.

You want users to explore the site still and go beyond the home page - but to your point I agree that the home page should make a first impression that makes users want to explore deeper into it.

**Comment by katomm** [comment]
Related: #357
